src/main/java/bookstore/validation/IsbnValidator.java

```diff
@@ -0,0 +1,12 @@
+package bookstore.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class IsbnValidator implements ConstraintValidator<Isbn, String> {
+
+    @Override
+    public boolean isValid(String s, ConstraintValidatorContext constraintValidatorContext) {
+        return s != null && s.length() == 13;
```

Comment:
such validation wil consider such string: `             ` as valid :)

it is better to use some regex